kale/predict/losses.py

```diff
@@ -27,6 +27,72 @@ def cross_entropy_logits(linear_output, label, weights=None):
     return loss, correct
 
 
+def topk_accuracy(output, target, topk=(1,)):
+    """Computes the precision@k for the specified values of k
+
+    Args:
+        output (Tensor): Generated predictions. Shape: [batch_size, class_count].
+        target (Tensor): Ground truth. Shape: [batch_size]
+        topk (tuple(int)): Compute accuracy at top-k for the values of k specified in this parameter.
+    Returns:
+        list(Tensor): A list of tensors of the same length as topk with boolean in.
```

Comment:
This can be made clearer. 
From the doc, I understand that the output is a list of tensors and the number of tensors is equal to the length of topk.
Please explain more clearly what each tensor is. "with boolean in" is too abstract. What is the size of each tensor? The number of samples? True means in top k and false means not?